projects/orangecrab/README.md

```diff
@@ -0,0 +1,48 @@
+# OrangeCrab Development Setup
+
+This sub-project contains some demo applications for targeting the [OrangeCrab FPGA r0.2.1](https://orangecrab-fpga.github.io/orangecrab-hardware/docs/r0.2.1) in combination with the OrangeCrab Pmod / Debug Expander board and an [Adafruit FT232H JTAG Debugger](https://learn.adafruit.com/adafruit-ft232h-breakout).
```

Comment:
I suspect the notion of a _sub-project_ refers to where the code originates. It doesn't seem apt here.